### ISSUE TITLE
Question: Do iOS 13 updates fix iOS issues ?

### ISSUE THREAD

**Issue by Aarbel** [eligrey/FileSaver.js#586]
Do iOS 13 updates fix iOS issues with FileSaver ?

Cf : https://bugs.webkit.org/show_bug.cgi?id=167341#c35

**Comment by zinglax** [comment]
Great question! 
Currently dealing with the blob opening in a new tab on iOS issue.  
Would love to just be able to download some JSON data from my web app seamlessly on iOS.

**Comment by Aarbel** [comment]
@zinglax, yes could be great that webkit APi opens native download manager on iOS.
It seems they released an update.

**Comment by theblang** [comment]
@Aarbel @zinglax I just updated to Xcode 11, which has an iOS 13 simulator. It does appear that `saveAs` works in iOS 13 mobile Safari! Albeit, I do see a console error: `Failed to load resource: Frame load interrupted blob:http://localhost:3000/2c925653-550c-457a-b063-a517bda49ce8`.

**Comment by rdelrioge** [comment]
any update on this?

**Comment by theblang** [comment]
@rdelrioge FWIW, we decided to just hide the download link for the file in question when on iOS Safari < 13. [This user agent parser lib](https://github.com/faisalman/ua-parser-js) helped a lot for checking that.

**Comment by DevJhns** [comment]
@zinglax @Aarbel 
any news, can I download/preview file from blobs in same tab, with native download manager as on ipad?

**Comment by camboui** [comment]
Hi, 

I've lost some time today with blob download on IOS. Like you, Safari was previewing it instead of downloading it with its file manager.
The trick is to remove the blob type : 
```
    const blob = new Blob([data]);
    saveAs(blob, fileName);
```
It shows the IOS download popup !
Hope it helps.
